src/docs/wtstats.dox

```diff
@@ -0,0 +1,33 @@
+/*! @page wtstats Visualizing performance with wtstats
+
+The WiredTiger distribution includes the \b wtstats tool that can be used to examine
+information generated using statistics logging (see @ref statistics_log).
+
+After running an application with statistics logging configured, the statistics output files will be in the database home directory.  By default, these are named \c WiredTigerStat.* .  In the database home directory, run this command, replacing \c ... with the path to the WiredTiger installation directory:
+\code{.sh}
+python .../tools/wtstats.py WiredTigerStat.*
```

Comment:
@sueloverso, perhaps we can mention `--help` rather than maintain the doc separately - I would think it's likely to get out of sync.  I agree we should cover using directory name instead of files.